site/en/r2/tutorials/distribute/keras.ipynb

```diff
@@ -1,25 +1,38 @@
 {
```

Comment:
<p>The link doesn't work.</p><p><br></p><p>https://github.com/tensorflow/docs/blob/master/site/en/r2/tutorials/distribute/training_loops.ipynb</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I believe it will work appropriately as published, as the source code has:</p><p><code>/tutorials/distribute/training_loops</code></p><p><br></p>